src/twisted/trial/runner.py

```diff
@@ -62,12 +61,8 @@ def isPackageDirectory(dirname):
     if dirname is a package directory.  Otherwise, returns False
     """
     def _getSuffixes():
-        if _PY3:
-            import importlib
-            return importlib.machinery.all_suffixes()
-        else:
-            import imp
-            return list(zip(*imp.get_suffixes()))[0]
+        import importlib
```

Comment:
these two importlib imports can move to the top (1)

Reply:
done